From f96b76efb810d7d559254d0ec58de628e09f525a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mart=C3=AD=20Bol=C3=ADvar?= <marti.f.bolivar@gmail.com>
Date: Mon, 13 Jan 2020 08:42:05 -0800
Subject: [PATCH] rule.py: fix missing comma
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

A line in the defined_keywords list is missing a comma. Add it.

Signed-off-by: Martí Bolívar <marti.f.bolivar@gmail.com>

Upstream-Status: Backport [https://github.com/Grokzen/pykwalify.git]
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pykwalify/rule.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pykwalify/rule.py b/pykwalify/rule.py
index 7ac2c9e..f044b69 100644
--- a/pykwalify/rule.py
+++ b/pykwalify/rule.py
@@ -340,7 +340,7 @@ class Rule(object):
             ('matching', 'matching'),
             ('matching_rule', 'matching_rule'),
             ('name', 'name'),
-            ('nullable', 'nullable')
+            ('nullable', 'nullable'),
             ('parent', 'parent'),
             ('pattern', 'pattern'),
             ('pattern_regexp', 'pattern_regexp'),
-- 
2.18.2

